app/utils/rxjs/getPacketLossStats$.ts

```diff
@@ -20,9 +20,32 @@ export function getPacketLossStats$(
 			let inboundPacketsLost = 0
 			let outboundPacketsSent = 0
 			let outboundPacketsLost = 0
+			let candidatePairID: string | undefined = undefined
+			let remoteCandidateID: string | undefined = undefined
+			let remoteAddress: string | undefined = undefined
 
 			newStatsReport.forEach((report) => {
 				const previous = previousStatsReport.get(report.id)
+
+				if (report.type === 'transport') {
+					candidatePairID = report.selectedCandidatePairId
+				}
+				if (candidatePairID) {
+					remoteCandidateID =
+						newStatsReport.get(candidatePairID)?.remoteCandidateId
+				}
+				if (remoteCandidateID) {
+					remoteAddress = newStatsReport.get(remoteCandidateID).address
+				}
+				if (remoteAddress !== undefined && remoteAddress !== '141.101.90.0') {
+					console.warn(
+						"PeerConnection doesn't appear to be connected to anycast 141.101.90.0"
+					)
+					if (!previous) {
```

Comment:
This will still alert every time the stats are checked (every second) — we should probably have a boolean before the `combineLatest` operator up above, so that users only see one alert.

Reply:
Got it. Fixed below.